sdk/python/v1beta1/kubeflow/katib/constants/constants.py

```diff
@@ -30,6 +30,24 @@
 
 DEFAULT_PRIMARY_CONTAINER_NAME = "training-container"
 
+# Label to identify Experiment's resources.
+EXPERIMENT_LABEL = "katib.kubeflow.org/experiment"
+
+# True means that Katib CR is in this condition.
+CONDITION_STATUS_TRUE = "True"
+
+# Experiment conditions.
+# TODO (andreyvelich): Use API enums when Katib SDK supports it.
+# Ref: https://github.com/kubeflow/katib/issues/1969.
+EXPERIMENT_CONDITION_CREATED = "Created"
+EXPERIMENT_CONDITION_RUNNING = "Running"
+EXPERIMENT_CONDITION_RESTARTING = "Restarting"
+EXPERIMENT_CONDITION_SUCCEEDED = "Succeeded"
+EXPERIMENT_CONDITION_FAILED = "Failed"
```

Comment:
Consider switching to use enums, e.g. `ExperimentCondition.Created`, `ExperimentCondition.Running`, etc.

Reply:
@terrytangyuan We are going to use enums from `model`, once this is fixed: https://github.com/kubeflow/katib/issues/1969.